system-parachains/asset-hubs/asset-hub-polkadot/src/staking/mod.rs

```diff
@@ -49,10 +130,14 @@ parameter_types! {
 type VoterBagsListInstance = pallet_bags_list::Instance1;
 impl pallet_bags_list::Config<VoterBagsListInstance> for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type ScoreProvider = nom_pools::StakingMock;
+	type ScoreProvider = Staking;
 	type WeightInfo = (); // TODO weights::pallet_bags_list::WeightInfo<Runtime>;
```

Comment:
how about generating weights for all staking related pallets? I am closing the rest in another PR.
we cannot leave this todos for longer